basics_of_sst-core/Example00/README.md

```diff
@@ -0,0 +1,8 @@
+Example project for creating a component for SST
+
+Created for SST-Core Version (9.1.0)  
+Also works with SST-Core v10.0.0
```

Comment:
ExampleComponent.cc says it works with v9-v11, this says v9-v10. Perhaps it would be good to pick one place to keep this info for each tutorial example.